### DISCUSSION TITLE
Example storefronts?

### DISCUSSION THREAD

**Discussion by Jarrodsz** [saleor/saleor#12030] [Q&A]
Very interested in using this insane good project in the near future.
Are there example storefronts? Any opensource storefronts that could be used as a foundation/reference point?

It would be great to see some storefronts in different styles.

**Comment by timuric** [comment]
Official open-source Saleor storefront:

https://github.com/saleor/react-storefront


On top of my head here are 3 examples that are quite different from each other:

https://www.prettygreen.com/

https://cargo.co/

https://www.colophon-foundry.org/

**Reply by Jarrodsz** [reply]
Are these opensource? I would not like to reinvent the wheel but guess mostly it will be react and components to build the storefronts and use some 'skeleton' components that you style. Nice list of examples! Wondered why no one asked about opensource store fronts before.

**Reply by timuric** [reply]
Saleor provides open-source storefront with checkout https://github.com/saleor/react-storefront
